package-support/sideboard-server.cfg

```diff
@@ -11,8 +11,12 @@ root = "INFO"
 [[syslog]]
 class = "logging.handlers.SysLogHandler"
 address = "/dev/log"
+formatter = syslog
 
 [formatters]
+[[syslog]]
+format = "$$(levelname)-5.5s $$(threadName)s [$$(name)s] $$(message)s"
+
```

Comment:
Here we add a syslog formatter to our default config - note that it's not actually used by default so this is a backwards-compatible change.

The main motivation here is that syslog already prepends timestamps to log messages, so we want a different formatter so that we're not wasting log output.  (Those of you familiar with the Splunk Cost Fallacy know that Splunk is really expensive so that kind of thing actually adds up to real dollars.)